### PR TITLE
make random affine matrix

### PR DESCRIPTION
**Pull request by rijobro**
create random affine matrix (will be required for inverse transform)

### CODE REVIEW

**Comment by wyli** [comment]
thanks, need some constraints about the invertibility?

**Comment by rijobro** [comment]
No, I don't use these for inverting anything, I just use it to write nifti images for the purposes of testing. I need them to have affines along the lines of:

```
 0, 1,  0, 0
-1, 0,  0, 0
 0, 0, -1, 0
 0, 0,  0, 1
```
At any rate, this is trivially invertible.

File: tests/utils.py [hunk 1]
```diff
@@ -151,6 +151,17 @@ def make_nifti_image(array, affine=None):
     return image_name
 
 
+def make_rand_affine(ndim: int = 3):
+    """Create random affine transformation (with values == -1, 0 or 1)."""
+    vals = np.random.choice([-1, 1], size=ndim)
```
Comment (wyli) [inline tests/utils.py:156]: would be great to support random generator e.g.: https://github.com/Project-MONAI/MONAI/blob/master/monai/data/synthetic.py#L51-L55

File: tests/utils.py [hunk 2]
```diff
@@ -151,6 +151,17 @@ def make_nifti_image(array, affine=None):
...
+    positions = np.random.choice([0, 1, 2], size=ndim, replace=False)
```
Comment (wyli) [inline tests/utils.py:157]: do you mean `[0, 1]` when ndim==2?

File: tests/utils.py (hunk 1 above)
Comment (rijobro) [reply tests/utils.py]: done!

File: tests/utils.py (hunk 2 above)
Comment (rijobro) [reply tests/utils.py]: thanks, solved with `range(ndim)`

**Review by wyli — APPROVED** [review]

**Pull request merged**